pkg/config/controller_manager.go

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) 2023 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package config
+
+import (
+	"k8s.io/client-go/tools/leaderelection/resourcelock"
+)
+
+// ControllerManagerConfig defaults
+const (
+	defaultMetricsAddr                        = ":8080"
+	defaultEnableLeaderElection               = false
+	defaultLeaderElectionID                   = "druid-leader-election"
+	defaultLeaderElectionResourceLock         = resourcelock.LeasesResourceLock
+	defaultIgnoreOperationAnnotation          = true
+	defaultDisableEtcdServiceAccountAutomount = false
+	defaultDisableLeaseCache                  = false
+)
+
+// ControllerManagerConfig contains configuration for the controller manager.
+type ControllerManagerConfig struct {
```

Comment:
These are command line flags and I would personally prefer to follow well accepted structure where we have a top level `cmd` directory instead where we have `LeaderElectionConfig` and likewise defined. `main.go` will anyways have a dependency to commands.